flow360/component/flow360_params/flow360_params.py

```diff
@@ -155,36 +155,42 @@ class MeshBoundary(Flow360BaseModel):
 class Boundary(Flow360BaseModel, metaclass=ABCMeta):
     """Basic Boundary class"""
 
-    type: str
+    model_type: str
     name: Optional[str] = pd.Field(
         None, title="Name", description="Optional unique name for boundary."
     )
 
+    # pylint: disable=missing-class-docstring,too-few-public-methods
+    class Config(Flow360BaseModel.Config):
+        deprecated_aliases = [DeprecatedAlias(name="model_type", deprecated="type")]
+
```

Comment:
I don't think this is a good change. Every user will get tens of warnings with every case

Reply:
This was implemented for consistency with other Flow360BaseModel discriminators after consulting Paweł, but we can revert for cases where this breaks compatibility with solver.

Reply:
Reverted